apps/designer/app/designer/features/style-panel/shared/property-name.tsx

```diff
@@ -1,39 +1,89 @@
-import { Flex, Label, Tooltip } from "@webstudio-is/design-system";
+import { useState } from "react";
+import type { StyleProperty } from "@webstudio-is/css-data";
+import {
+  Button,
+  Flex,
+  Box,
+  Text,
+  Label,
+  Tooltip,
+  Popover,
+  PopoverContent,
+  PopoverPortal,
+  PopoverTrigger,
+  Separator,
+} from "@webstudio-is/design-system";
+import { UndoIcon } from "@webstudio-is/icons";
 import { useIsFromCurrentBreakpoint } from "./use-is-from-current-breakpoint";
-import { propertyNameColorForSelectedBreakpoint } from "./constants";
-import type { PropertyProps } from "../style-sections";
 
-export const PropertyName = ({ property, label, css }: PropertyProps) => {
+type PropertyProps = {
+  property: StyleProperty | StyleProperty[];
+  label: string;
+  onReset: () => void;
+};
+
+export const PropertyName = ({ property, label, onReset }: PropertyProps) => {
   const isCurrentBreakpoint = useIsFromCurrentBreakpoint(property);
+  const [isOpen, setIsOpen] = useState(false);
+  const isPopoverEnabled = isCurrentBreakpoint;
```

Comment:
Since we use "set" keyword in the design system, maybe we should call it isSet and the other options isCascaded and isInherited? If you agree I would rename it all including that function

Reply:
lets add `isFeatureEnabled('reset')` and add reset = false to feature-flags/flags so that its not on production for now until we get the design

on preview we can enable it by adding "reset" to the FEATURES env variable, same locally in .env

I am wondering if we should just have an * in preview to avoid the need to add and remove new flags in env and just always have all things enabled in preview

Reply:
Where FEATURES env passed from?

Reply:
environment variable, added asterix support here https://github.com/webstudio-is/webstudio-designer/pull/586

Reply:
will enable asterix on preview, so we never have to enable them manually there

Reply:
set it to asterix locally too, for convenience

Reply:
environment variable is used to override, but flags.ts file is used to set the default